x/pairing/client/cli/tx_stake_provider.go

```diff
@@ -25,19 +25,21 @@ var _ = strconv.Itoa(0)
 
 func CmdStakeProvider() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "stake-provider [chain-id] [amount] [endpoint endpoint ...] [geolocation] --from <address> --provider-moniker <moniker>",
+		Use:   "stake-provider [chain-id] [amount] [endpoint endpoint ...] [geolocation] {validator} --from <address> --provider-moniker <moniker>",
 		Short: `stake a provider on the lava blockchain on a specific specification`,
 		Long: `args:
 		[chain-id] is the spec the provider wishes to support
-		[amount] is the ulava amount to be staked
+		[amount] is the ulava amount to be staked, this is the final amount, if is lower than the the original stake it will unbond from the validator
```

Comment:
```suggestion
		[amount] is the ulava amount to be staked, this is the final amount, if is lower than the the original stake it will unbond from the provider (and delegated validators)
```